openslides/core/static/templates/core/manage-projectors.html

```diff
@@ -82,19 +83,30 @@ <h1 translate>Manage projectors</h1>
         <div>
           <label for="resolution{{ projector.id }}" class="control-label"><translate>Resolution</translate>:</label>
           <div id="resolution{{ projector.id }}">
-            <input ng-model="resolutions[projector.id].width"
-                   ng-model-option="{debounce: 2000}"
-                   ng-change="changeResolution(projector.id)"
-                   class="form-control resolution"
-                   id="{{ projector.id }}_width"
-                   type="number">
-            x
-            <input ng-model="resolutions[projector.id].height"
-                   ng-model-option="{debounce: 2000}"
-                   ng-change="changeResolution(projector.id)"
-                   class="form-control resolution"
-                   id="{{ projector.id }}_height"
-                   type="number">
+            <div>
+              <span>
+                {{ resolutions[projector.id].width }}&times;{{ resolutions[projector.id].height }}
+              </span>
+              <div class="dropdown" uib-dropdown>
+                <button class="btn btn-default btn-sm" id="aspectRatio{{ projector.id }}" uib-dropdown-toggle>
+                  <translate>Aspect ratio</translate>: {{ resolutions[projector.id].aspectRatio }}
+                  <span class="caret"></span>
+                </button>
+                <ul class="dropdown-menu" uib-dropdown-menu aria-labelledby="aspectRatio{{ projector.id }}">
+                  <li ng-repeat="(aspectRatio, value) in aspectRatios track by $index">
+                    <a href ng-click="setAspectRatio(projector, aspectRatio); saveResolution(projector);">
+                      <i class="fa fa-check" ng-if="aspectRatio === resolutions[projector.id].aspectRatio"></i>
+                      {{ aspectRatio }}
+                    </a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+            <div>
+              <rzslider
```

Comment:
suggestion/wish: add a custom css class to use our nicer OpenSlides corporate blue for slider color.